contracts/acl/IACLOracle.sol

```diff
@@ -6,5 +6,5 @@ pragma solidity ^0.4.24;
 
 
 interface IACLOracle {
-    function canPerform(address who, address where, bytes32 what, uint256[] how) external view returns (bool);
+    function canPerform(address user, address who, address where, bytes32 what, uint256[] how) external view returns (bool);
```

Comment:
Might be worth adding some comments for the parameters, since `user` and `who` are easily mixed up :).